src/sentry/identity/gitlab/provider.py

```diff
@@ -1,11 +1,15 @@
 from __future__ import absolute_import
 
+import logging
+
 from sentry import http
 from sentry.auth.exceptions import IdentityNotValid
 from sentry.http import safe_urlopen, safe_urlread
 from sentry.identity.oauth2 import OAuth2Provider
 from sentry.utils import json
 
+logger = logging.getLogger('sentry.integrations.gitlab')
```

Comment:
Oh just found out the base class has a logger. Going to use that.

Reply:
Making a logger instance in each module is 👌 and simpler than adding more imports.

Reply:
Ironically enough in this case, the `get_user_info` method is not a part of the provider, so I have to import logger here.